Dockerfile

```diff
@@ -14,6 +14,10 @@ ENV PYTHONFAULTHANDLER=1 \
   POETRY_VERSION=1.4.1
 
 RUN pip install "poetry==$POETRY_VERSION"
+RUN pip install streamlit
```

Comment:
The project is no longer using streamlit for the web-ui so there shouldn't be a need to add streamlit